src/api/PacbioRun.js

```diff
@@ -1,41 +1,19 @@
 import handlePromise from './PromiseHelper'
 
-const NO_OF_COLUMNS = 12
-const NO_OF_ROWS = 8
-
-const columns = () => {
-  return Array.from(Array(NO_OF_COLUMNS), (e, i) => String(i + 1))
-}
-
-const rows = () => {
-  return Array.from(Array(NO_OF_ROWS), (e, i) => String.fromCharCode(65 + i))
-}
-
-const buildWell = (row, column) => {
+const buildWell = (row, column, generateHiFiDefault='', preExtenstionTimeDefault='2') => {
```

Comment:
you can turn this into an object literal which negates the need to have it as a function. I'm not sure that using generateHiFiDefault is the best name as it is an argument with a default. If you make it the same as the attribute then you can use the spread operator e.g.

```
const buildWell = (row, column, generate_hifi='', pre_extension_time='2') => ({
  row, 
  column,
  position: `${row}${column}`,
  movie_time: '',
  insert_size: '',
  on_plate_loading_concentration: '',
  sequencing_mode: '',
  generate_hifi,
  libraries: [],
  pre_extension_time
})
```